storyblok/storyblok-js-client
# `INT-411` - Feat/int 411 typescript

## Pull request type

Jira Link: [INT-411](https://storyblok.atlassian.net/browse/INT-411)




## How to test this PR


Just proceed to install the application as before. Your current code will probably starts to break, because you're implementing it without types. You can check the type declarations by hovering over the cause if the problem. Some TypeScript knowledge might be necessary.

## What is the new behavior?



- Every file was refactored with TypeScript 


## Other information
